### PR TITLE
[MIG][12.0] Merge odoo-docker-template 

### PR DESCRIPTION
**Pull request by JevinD**
Don't merge until Docker repo changes has been merged

### CODE REVIEW

**Comment by JevinD** [comment]
@ursa-ci Run tests

File: README.md [hunk 1]
```diff
@@ -103,3 +91,7 @@ Examples:
 
 Report any issue to this
 [Github project](https://github.com/ursais/odoo-template/issues).
+
+## Roadmap
+
+* Push the production database to the other environments
```
Comment (max3903) [inline README.md]: @JevinD That is done. Not in the roadmap anymore...

Comment (JevinD) [reply README.md]: @max3903 I updated with my next objective for documentation. Is there any other plans to include?

**Comment by JevinD** [comment]
@ursa-ci Run tests

**Comment by max3903** [comment]
@ursa-ci Run tests

**Comment by max3903** [comment]
@ursa-ci Run tests

**Comment by max3903** [comment]
@ursa-ci Run tests

**Comment by max3903** [comment]
@ursa-ci Run tests

**Comment by max3903** [comment]
@JevinD Can you check the jenkins error?

**Comment by JevinD** [comment]
@max3903 This issue is coming from core odoo. The function is registering b"   " + content as just content.
https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/tests/test_mimetypes.py#L69

Can we review?

**Comment by max3903** [comment]
@JevinD Is it working on Odoo's runbot?

**Pull request closed without merging**